db/seeds.rb

```diff
@@ -10,9 +10,37 @@
 fe = Specialty.create(name: "Front End", url_name: "front_end", image: File.new("#{Rails.root}/app/assets/images/back_end/Wolf_animal_totem.png"))
 be = Specialty.create(name: "Back End", url_name: "back_end", image: File.new("#{Rails.root}/app/assets/images/front_end/hawk_animal_totem.png"))
 
-Developer.create(name: "JP", last_name: "Perry", email: "hotdog1@hotmail.com", password: "$", rate: 10, specialty_id: db.id, skills_rating: 2 )
-Developer.create(name: "Steve", last_name: "Pentler", email: "hotdog2@hotmail.com", password: "$", rate: 20, specialty_id: fe.id, skills_rating: 3 )
-Developer.create(name: "Hector", last_name: "Huertas", email: "hotdog3@hotmail.com", password: "$", rate: 30, specialty_id: be.id, skills_rating: 4 )
-Developer.create(name: "Aaron", last_name: "Greenspan", email: "hotdog4@hotmail.com", password: "$", rate: 40, specialty_id: be.id )
+Developer.create(name: "JP", 
+                last_name: "Perry",
+                email: "hotdog1@hotmail.com",
+                password: "$",
+                rate: 10,
+                specialty: db,
+                skills_rating: 2 )
+Developer.create(name: "Steve",
+                last_name: "Pentler",
+                email: "hotdog2@hotmail.com",
+                password: "$",
+                rate: 20,
+                specialty: fe,
+                skills_rating: 3 )
+Developer.create(name: "Hector",
+                last_name: "Huertas",
+                email: "hotdog3@hotmail.com",
+                password: "$",
+                rate: 30,
+                specialty: be,
+                skills_rating: 4 )
+Developer.create(name: "Aaron",
+                last_name: "Greenspan",
+                email: "hotdog4@hotmail.com",
+                password: "$",
+                rate: 40,
+                specialty: be )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.